source/MRViewer/MRToolbar.cpp

```diff
@@ -12,8 +12,8 @@
 
 namespace MR
 {
-ImVec2 vecToImVec( const Vector2f& vec ) { return ImVec2( vec.x, vec.y ); };
-Vector2f imVecToVec( const ImVec2& vec ) { return Vector2f( vec.x, vec.y ); };
+ImVec2 vecToImVec( const Vector2f& vec ) { return ImVec2( vec.x, vec.y ); }
+Vector2f imVecToVec( const ImVec2& vec ) { return Vector2f( vec.x, vec.y ); }
```

Comment:
we have imlicit casts now, now need in these functions

Reply:
removed